Makefile

```diff
@@ -18,6 +18,10 @@ build/sentry_test_unit: build
 test: update-test-discovery test-integration
 .PHONY: test
 
+test-unit: update-test-discovery build/sentry_test_unit
+	./build/sentry_test_unit
```

Comment:
on my machine this still spits out an absurd number of errors that look vaguely like

```
[71087:13339984:20211208,163001.841338:WARNING crash_report_database_mac.mm:767] Failed to read report metadata for .../sentry-native/.sentry-native/completed/49b5cb3a-64f0-4d5f-86fb-ed731f0981df.meta
```

i've tried negative grepping and awking this away to no luck; i would imagine in the worst case scenario this would require some sort of custom build that changes the logging level on crashpad to error at minimum. any help here would be much appreciated as the unit test results are unnecessarily difficult to parse through by default thanks to my poor shell skills.

Reply:
You can ignore this. Just `rm -rf .sentry-native`, as that directory is safe to prune.
However now I think that we might want to have a solution for this, as I bet customers are hitting that as well sometimes and its not as simple to tell them "just remove that directory".

Reply:
i'll leave this as is for the time being. if a contributor ends up bumping into this problem we can loop back and look into addressing this then. the solutions i've explored so far to deal with this are all fairly ugly, so i'd rather take the hit and just manually prune the reports stored in my local DB.